sympy/series/limits.py

```diff
@@ -231,12 +231,11 @@ def doit(self, **hints):
                                     "infinity are not implemented")
 
         if z0.is_infinite:
-            if z0 not in (S.Infinity, S.NegativeInfinity):
-                cdir = sign(z0).simplify()
-                cdir = cdir/abs(cdir)
-                e = e.subs(z, cdir*z)
-                dir = "-"
-                z0 = S.Infinity
+            cdir = sign(z0).simplify()
```

Comment:
simplify() is fairly expensive. Would a cheaper method suffice?

Reply:
Yeah I put that just in case ..... we might not be needing it as of now though as we currently only supported fixed points and not variables ones through the following
```
        if(z0.has(z)):
            raise NotImplementedError("Limits approaching a variable point are"
                    " not supported (%s -> %s)" % (z, z0))
```
So yeah as we might not need a simplification for that as of now , I'll get rid of the `simplify` there.